packages/ui/v2/core/form/select/components.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+  components as reactSelectComponents,
+  ControlProps,
+  GroupBase,
+  InputProps,
+  MenuListProps,
+  MenuProps,
+  OptionProps,
+  SingleValueProps,
+  ValueContainerProps,
+} from "react-select";
+
+import { classNames } from "@calcom/lib";
+import { Icon } from "@calcom/ui/Icon";
+
+export const InputComponent = <
```

Comment:
I'm not happy with all the generics defined in this file, but otherwise react-select wouldn't stop complaining. Tried to abstract it into a function, but that would make it a lot more complicated for a few lines less code.. If anyone has a great idea — always welcome!